src/main/java/com/stuypulse/robot/commands/ChuteIntakeCommand.java

```diff
@@ -0,0 +1,57 @@
+package com.stuypulse.robot.commands;
+
+import com.stuypulse.robot.RobotContainer;
+import com.stuypulse.robot.subsystems.Chute;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ChuteIntakeCommand extends CommandBase {
+
+    private Chute chute;
+    private static boolean isIntake;
+    private boolean previousValue;
+
+    public ChuteIntakeCommand(Chute chute) {
+        this.chute = chute;
+        isIntake = false;
+        previousValue = false;
+
+        // Use addRequirements() here to declare subsystem dependencies.
+        addRequirements(chute);
+    }
+
+    @Override
+    public void initialize() {
+
+    }
+
+    @Override
+    public void execute() {
+        if (previousValue != RobotContainer.gamepad.getRawDPadDown()) {
```

Comment:
Why do you need this part? OI (configuring commands to buttons) should already do this for you